custom_components/tuya_local/devices/aktobis_wdh_310ek_dehumidifier.yaml

```diff
@@ -0,0 +1,175 @@
+name: Aktobis WDH-310EK dehumidifier
+products:
+  - id: bf43c599b40eada03aivfh
+    name: WDH-310EK
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
```

Comment:
In general, setting icons explicitly is discouraged, and should not be necessary for these two.